### DISCUSSION TITLE
Start Menu Bar Is Entirely Black

### DISCUSSION THREAD

**Discussion by jpjc4785** [valinet/ExplorerPatcher#2107] [Q&A]
Hello,

I have a very odd issue in that my entire start menu area is all black. I can't read any text or modify it to not be black. Unless there is a setting that I am missing. I have tried changing the color using Windows and that has not helped. I would appreciate any help that can be offered.
![Black Start Menu](https://github.com/valinet/ExplorerPatcher/assets/145616150/d8441130-da84-4158-b19e-ec967df6dda2)

**Comment by jpjc4785** [comment]
It’s a program called open shell. It doesn’t have anything color related and just allows you to customize the start menu. I had both installed previously prior to the last update and no issues but when I had to reformat that picture was the result.Joel JohnsonSent from my iPhoneOn Sep 20, 2023, at 5:23 PM, Poopooracoocoo ***@***.***> wrote:﻿
Saw that your start icon is different. :) How are you customising the appearance of the taskbar?

**Reply by Poopooracoocoo** [reply]
Do you see this without Open Shell?

**Comment by pyrates999** [comment]
You might have classic shell installed.  Uninstall it if you do and install open shell instead.
